packages/shared/src/utils/helpers/chart-barrier-store.ts

```diff
@@ -32,7 +34,7 @@ export class ChartBarrierStore {
         high_barrier?: string | number,
         low_barrier?: string | number,
         onChartBarrierChange: TOnChartBarrierChange = null,
-        { color, line_style, not_draggable }: TChartBarrierStoreOptions = {}
+        { color, hideBarrierLine, line_style, not_draggable, shade }: TChartBarrierStoreOptions = {}
```

Comment:
Are you planning to add sanity check for other barriers (not only for Multipliers)? Or there is no need and such things as shades will work as before?

Reply:
the shades will work as before. the lines that are updated are the result of combining 2 copies of the same file, and putting it into @shared :) . Although adding a sanity check for some other barriers is a good idea, thank you)